app/src/main/java/com/cornellappdev/volume/ui/screens/PublicationScreen.kt

```diff
@@ -0,0 +1,167 @@
+package com.cornellappdev.volume.ui.screens
+
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.apollographql.apollo3.api.BooleanExpression
+import com.cornellappdev.volume.R
+import com.cornellappdev.volume.analytics.EventType
+import com.cornellappdev.volume.analytics.NavigationSource
+import com.cornellappdev.volume.analytics.VolumeEvent
+import com.cornellappdev.volume.data.models.Article
+import com.cornellappdev.volume.data.models.Publication
+import com.cornellappdev.volume.ui.components.general.CreateFollowPublicationRow
+import com.cornellappdev.volume.ui.components.general.CreateHorizontalPublicationRow
+import com.cornellappdev.volume.ui.components.onboarding.isScrolledToTheEnd
+import com.cornellappdev.volume.ui.states.PublicationsRetrievalState
+import com.cornellappdev.volume.ui.theme.VolumeOffWhite
+import com.cornellappdev.volume.ui.theme.VolumeOrange
+import com.cornellappdev.volume.ui.theme.notoserif
+import com.cornellappdev.volume.ui.viewmodels.PublicationsViewModel
+
+@Composable
+fun PublicationsScreen(
```

Comment:
Rename this file back to PublicationsScreen since it has this composable inside of it